content/gophercon-2015/gophercon-friday-hack-day.md

```diff
@@ -0,0 +1,58 @@
++++
+author = ["Dave Cheney"]
+date = "2015-06-14"
+linktitle = "GopherCon Friday Hack Day details"
+title = "GopherCon Friday Hack Day details"
++++
+With less than a month til GopherCon 2015 we’ve been hard at work on plans for the Friday 10th Hack Day. The previous year’s unofficial Friday Hack Day was so popular we decided this year to integrate it into the conference proper. 
+
+We're providing the space and the wifi, you bring your favorite project and work together with friends you only get to see in person once a year at GopherCon.
+
+After the brisk start of the two previous days, Friday will arrive at a more leisurely pace with most activities starting at 10 am.
+
+Kicking off in the Mile High Ballroom theatre we will feature a presentation “Investment 101 for Startups” by Robin Vasan, managing director a Mayfield, one of the most prestigious investment firms in the country. Robin will be joined for a question and answer session by the CEO's from two of Mayfield's investments. Paul Dix from InfluxDB and Mitchell Hashimoto from HashiCorp who will share their experiences as founders going through the investment process.
```

Comment:
Fixed. I have no idea how I can make so many errors when I just copied and pasted the text from the website.